sdk/purview/azure-analytics-purview-workflow/README.md

```diff
@@ -0,0 +1,63 @@
+# Azure PurviewWorkflow client library for Java
+
+Azure PurviewWorkflow client library for Java.
+
+This package contains Microsoft Azure PurviewWorkflow client library.
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][docs]
+- [Product documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-analytics-purview-workflow;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-analytics-purview-workflow</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authentication
+
+[Azure Identity][azure_identity] package provides the default implementation for authenticating the client.
+
+## Key concepts
```

Comment:
We need to fill these sections and make getting started easy for this library before we preview.

Reply:
Fixed, thanks